docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst

```diff
@@ -54,6 +54,24 @@ The launcher has the following additional requirements on the installation node:
 -  Access to the Slurm or PBS command line interface for the cluster
 -  Access to a cluster-wide file system with a consistent path names across the cluster
 
+.. _proxy-config-requirements:
+
+**********************************
+ Proxy Configuration Requirements
+**********************************
+
+If internet connectivity requires a use of a proxy, verify the following requirements:
+
+-  Ensure that the proxy variables are defined in `/etc/environment` (or `/etc/sysconfig/proxy` on
+   SLES).
+-  Ensure that login, admin, and compute nodes, which resolve to short names, are included in the
```

Comment:
This sentence is difficult to parse.    Also it seems untenable to list all the compute nodes in the no_proxy list -- we don't define them on our test systems, so Determined apparently doesn't communicate between the compute nodes?   I would minimally separate out login/admin to be more prescriptive, and then add compute nodes separately (if we need to do so).

- Ensure that the `no_proxy` setting covers the login and admin nodes.   If these nodes may be referenced by short names known only within the cluster, they must explicitly be included in the `no_proxy` setting.
- If your experiment code communicates between compute nodes with a protocol that honors proxy environment variables, you should additionally include the names of all compute nodes in the `no_proxy` variable setting.